rtl/nbcd/nbcd.cpp

```diff
@@ -168,9 +168,9 @@ Decimal & Decimal::divide(const Decimal & other)
     lsb = 0;
     msb = hi >= maxDigits ? maxDigits-1 : hi;
 
-    const byte spare = 2;
+    const byte spare = 3;
```

Comment:
Spare is now 3 because it is effectively performing another iteration of the division to determine what the next significant digit is.  That involves looking at an extra digit in the numerator - so we need an extra 0 on the end (or front since this is little endian.)